aiocogeo/cog.py

```diff
@@ -46,11 +46,11 @@ async def read(
         ...
 
     @abc.abstractmethod
-    async def point(self, x: Union[float, int], y: Union[float, int]) -> Union[Union[np.ndarray, np.ma.masked_array], List[Union[np.ndarray, np.ma.masked_array]]]:
+    async def read_point(self, x: Union[float, int], y: Union[float, int]) -> Union[Union[np.ndarray, np.ma.masked_array], List[Union[np.ndarray, np.ma.masked_array]]]:
```

Comment:
to be able to use rio-tiler baseclass we can't have point nor preview in the COGReader class ;-)

Reply:
These were added to COGReader before a rio-tiler integration was on my radar, I only added them because I was bored one night and liked the functionality :).  So I think I'd rather just remove these from COGReader, keep it low level, and include these methods exclusively on this class.